docs/example.md

```diff
@@ -0,0 +1,27 @@
+```
+      $(document).ready(function () {
+            $("#Stage").ontology_search({});
+            $("#Stage").ontology_search("updateOptions", {filter: ["Developmental_stage%20AND%20Homo_sapiens"]});
```

Comment:
Filters and/or issue fixed on main branch. filter: ["Developmental_stage", "Homo_sapiens"] should be working